tools/nl/embeddings/data/curated_input/main/sdg_topics.csv

```diff
@@ -1,6 +1,6 @@
 dcid,Name,Description,Override_Alternatives,Curated_Alternatives
 dc/topic/sdg,Sustainable Development Goals,,,sdg;sdg;sdg;sdg;sdg Sustainable Development Goals;sustainable development goal Sustainable Development Goals
-dc/topic/sdg_1,SDG No Poverty,,,sdg 1;sdg 1;sdg1;sdg 1;sdg No Poverty;sustainable development goal No Poverty;
+dc/topic/sdg_1,SDG No Poverty,,,sdg 1;sdg 1;sdg1;sdg 1;sdg No Poverty;sustainable development goal No Poverty;End poverty
```

Comment:
curious where 'End poverty' came from and should we add 'sdg' prefix to it

Reply:
This is from dc/topic/SDG_1 that is removed in main_topics.csv